datadog_lambda/wrapper.py

```diff
@@ -150,6 +153,7 @@ def __call__(self, event, context, **kwargs):
             self._after(event, context)
 
     def _before(self, event, context):
+        self.response = None
```

Comment:
let's put this line inside the `try:...` block, so we avoid datadog code crashing customer application as much as we can....that is, datadog should fail quietly (only log something) but no interruption to customer application. I know this line is pretty safe, but it may "invite" future developers to add more lines outside the try block because they saw some lines outside...